### ISSUE TITLE
Issue with optimizing execution results

### ISSUE THREAD

**Issue by Timbanin-s** [lime-j/RDNet#15]
![Image](https://github.com/user-attachments/assets/615c90e5-b8c1-4ee0-82af-ce034659bae8)

Thank you for your team's work! Based on my current run, the result looks like what's shown in the image. Could you please let me know if there might be something I misconfigured?

**Comment by lime-j** [comment]
Maybe a small learning rate I guess?

**Comment by Timbanin-s** [comment]
Yes, I did not perform any training. I simply downloaded the following pre-trained models and performed inference. 
cls_model.pth
focal.pth
rdnet.pth

Please tell me what the correct procedure should be.
